examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
 
 # Persistent Installation of MySQL and WordPress on Kubernetes
 
-This example describes how to run a persistent installation of [Wordpress](https://wordpress.org/) using the [volumes](../../docs/user-guide/volumes.md) feature of Kubernetes, and [Google Compute Engine](https://cloud.google.com/compute/docs/disks) [persistent disks](../../docs/user-guide/volumes.md#gcepersistentdisk).
-
-We'll use the [mysql](https://registry.hub.docker.com/_/mysql/) and [wordpress](https://registry.hub.docker.com/_/wordpress/) official [Docker](https://www.docker.com/) images for this installation. (The wordpress image includes an Apache server).
-
-We'll create two Kubernetes [pods](../../docs/user-guide/pods.md) to run mysql and wordpress, both with associated persistent disks, then set up a Kubernetes [service](../../docs/user-guide/services.md) to front each pod.
-
-This example demonstrates several useful things, including: how to set up and use persistent disks with Kubernetes pods; how to define Kubernetes services to leverage docker-links-compatible service environment variables; and use of an external load balancer to expose the wordpress service externally and make it transparent to the user if the wordpress pod moves to a different cluster node.
-
-## Get started on Google Compute Engine (GCE)
-
-Because we're using the `GCEPersistentDisk` type of volume for persistent storage, this example is only applicable to [Google Compute Engine](https://cloud.google.com/compute/). Take a look at the [volumes documentation](../../docs/user-guide/volumes.md) for other options.
-
-First, if you have not already done so:
-
-1. [Create](https://cloud.google.com/compute/docs/quickstart) a [Google Cloud Platform](https://cloud.google.com/) project.
-2. [Enable billing](https://developers.google.com/console/help/new/#billing).
-3. Install the [gcloud SDK](https://cloud.google.com/sdk/).
-
-Authenticate with gcloud and set the gcloud default project name to point to the project you want to use for your Kubernetes cluster:
-
-```sh
-gcloud auth login
-gcloud config set project <project-name>
+This example describes how to run a persistent installation of
+[WordPress](https://wordpress.org/) on Kubernetes, using
+[Persistent Volumes](../../docs/user-guide/persistent-volumes.md) and
+[Secrets](../../docs/user-guide/secrets.md).
+
+We'll use the [mysql](https://registry.hub.docker.com/_/mysql/) and
```

Comment:
Is there (or could there be) an image on GCR? Docker hub isn't very reliable.
https://github.com/kubernetes/kubernetes/issues/15251